conf/test.env

```diff
@@ -15,7 +15,7 @@ CONSUMER_COUNT=8
 
 KAFKA_ADDRESS=platform:9092
 KAFKA_GROUP=patchman
-UPLOAD_TOPIC=platform.upload.available
+UPLOAD_TOPIC=platform.inventory.host-egress
```

Comment:
I we don't depend on inventory API, we can remove INVENTORY_ADDRESS from confs listener and evaluator_common. We can also remove inventory.go module from our platform mock.

Reply:
We can also remove exposed 9001 port in docker-compose.

Reply:
We can't we use it for VMaaS & RBAC mocking